cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/ArtifactSelectorConfig.java

```diff
@@ -85,4 +87,9 @@ public String toString() {
       ", version='" + version + '\'' +
       '}';
   }
+
+  public io.cdap.cdap.api.artifact.ArtifactScope getApiArtifactScope() {
```

Comment:
don't think this should exist here either. The caller should make sure they handle the null scope case.